cdapython/constantVariables.py

```diff
@@ -1,11 +1,15 @@
 # Versioning Year Month Day of last push
-VERSION = "2022.1.24"
-DATABASETABLE_VERSION = "all_v2_1"
-DATABASETABLE = "gdc-bq-sample.integration"
-CDA_API_URL_ENV = "https://cda.cda-dev.broadinstitute.org"
+VERSION: str = "2022.3.21"
+DATABASETABLE_VERSION: str = "all_v3_0_subjects_meta"
+DATABASETABLE: str = "gdc-bq-sample.dev"
```

Comment:
Are we continuing to use gdc-bq-sample as the project or are we switching it all to broad-cda-dev? It doesn't matter much to me but I just want to try to put dev and integration under a similarly named project and dataset and stuff

Reply:
for integration we are using these tables but when we release to Maine I will switch them